code/datums/supplypacks.dm

```diff
@@ -228,6 +228,13 @@ GLOBAL_LIST_INIT(all_supply_groups, list(SUPPLY_EMERGENCY,SUPPLY_SECURITY,SUPPLY
 	contains = list(/obj/item/vending_refill/security)
 	containername = "SecTech supply crate"
 
+/datum/supply_packs/security/vending/clothingvendor
+	name = "Security Clothing Vendors Crate"
+	cost = 40
+	contains = list(/obj/item/vending_refill/secdrobe,
+					/obj/item/vending_refill/detdrobe)
+	containername = "security clothing vendor crate"
```

Comment:
The "Security Clothing Crate" and the "Security Clothing Vendors Crate" are a bit ambiguous when you search for them ingame, and I can imagine people might get confused by it:
![image](https://user-images.githubusercontent.com/57483089/128428864-6a45ca2e-08fd-4367-aa19-558704c8f05e.png)
You can check the contents of course, but it *might* be better if these resupply crates were renamed to "ThingDrobe Vendor Crate" or something instead. It's fine either way though.

Reply:
I _myself_ don't really think its all that confusing, and I having a separate crate for every vendor would clutter the console a bit too much.